src/handler.py

```diff
@@ -146,28 +143,25 @@ def newJob(event, context):
     timestamp_ms = ulid_obj.timestamp().int
 
     # Check that all required keys are present.
-    required_keys = ['site', 'device', 'instance', 'action', 'user_name', 'user_id', 'optional_params', 'required_params']
+    required_keys = ['site', 'device', 'instance', 'action', 'user_name', 
+                     'user_id', 'optional_params', 'required_params']
```

Comment:
For something like this where you're de-serializing JSON data and validating it, I would recommend a library like Marshmallow or Cerberus.

Strictly for validation, Cerberus looks good https://docs.python-cerberus.org/en/stable/usage.html

If you are wanting to validate and/or de-serialize JSON into an Python object or vice-versa, I have had good luck with Marshmallow: https://marshmallow.readthedocs.io/en/stable/

It's really useful to have a defined schema for requests, both for the users' and the developers' sanity. Using these sorts of libraries really forces good practices (e.g. defining a schema) and also some nice validation too.